meshroom/ui/qml/GraphEditor/GraphEditor.qml

```diff
@@ -455,13 +455,51 @@ Item {
                         computeRequest(nodeMenu.currentNode)
                     }
                 }
+                MenuItem {
+                    text: "Compute Selected Node(s)"
+                    visible: uigraph.selectedNodes.count > 0
+                    height: visible ? implicitHeight : 0
+
+                    enabled: {
+                        var canCompute = false
+                        for (var i = 0; i < uigraph.selectedNodes.count; ++i) {
+                            if (uigraph.graph.canCompute(uigraph.selectedNodes.at(i)) && uigraph.graph.canSubmitOrCompute(uigraph.selectedNodes.at(i))%2 == 1){
+                                canCompute = true
+                            }
+                        }
+                        return canCompute
+                    }
+
+                    onTriggered: {
+                        computeRequest(uigraph.selectedNodes)
+                    }
+                }
```

Comment:
There is no need to add a second menu to compute/submit the selected nodes. Everything that's done with these menus should instead be done with the regular "Compute" and "Submit" to avoid confusions for the user.